src/index.html

```diff
@@ -5,8 +5,278 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="container">
+      <nav class="container__nav-block">
+
+        <a href="#" class="container__nav-link is-active">Apple</a>
+        <a href="#" class="container__nav-link">Samsung</a>
+        <a href="#" class="container__nav-link">Смартфоны и телефоны</a>
+        <a href="#" class="container__nav-link"
+         data-qa="hover"
+         >Ноутбуки и компьютеры
+        </a>
```

Comment:
```suggestion
        <a 
          href="#"
          class="container__nav-link"
          data-qa="hover"
         >
           Ноутбуки и компьютеры
        </a>
```
Форматируй теги таким образом